scripts/quarantine_integration.yaml

```diff
@@ -445,6 +445,8 @@
     - fast_pair.storage.factory_reset.9keys
     - fast_pair.storage.factory_reset.default
     - fast_pair.storage.factory_reset.ak_minimal_backend
+    - nrf54l15dk/nrf54l10/cpuapp
```

Comment:
This seems wrong

Reply:
We should only disable `fast_pair.storage.factory_reset.default` and `fast_pair.storage.factory_reset.ak_minimal_backend`

Reply:
Should we actually quarantine a test even if it does not support a given board?